java/core/src/main/java/co/worklytics/psoxy/rules/github/PrebuiltSanitizerRules.java

```diff
@@ -111,6 +111,7 @@ public class PrebuiltSanitizerRules {
             .transform(Transform.Pseudonymize.builder()
                     .jsonPath("$..nameId")
                     .jsonPath("$..email")
+                    .jsonPath("$..value")
```

Comment:
🤔 this doesn't seem great; this will pseudsonymize any field named `value`, anywhere in the response. make it `$..emails[].value` or something like that? 

(not sure that's exactly correct JSONPath; but idea is .value fields below emails[] array)